package.json

```diff
@@ -46,7 +46,7 @@
 		"typescript": "^5.5.2"
 	},
 	"peerDependencies": {
-		"@rsbuild/core": "1.x || ^1.0.1-beta.0"
+		"@rsbuild/core": "1.x"
```

Comment:
I think we can remove `^1.0.1-beta.0` in the future, after more users have upgraded to v1 stable.

![image](https://github.com/user-attachments/assets/e6cceeb8-19dd-48bd-ae3b-e346089b1b4f)